style.css

```diff
@@ -0,0 +1,192 @@
+body {
+    background-color: #eee8aa;
+    font-family: 'Work Sans', sans-serif;
+    font-size: 14px;
+    color: #1e90ff;
+    height: 100%;
+}
+
+h1 {
+    font-size: 28px;
+    font-style: normal;
+    text-align: center;
+    letter-spacing: 2px;
+}
+
+
+.best-player-cards {
+    margin-top: 20px;
+    margin-bottom: 80px;
+}
+
+.best_players_cards__header {
+    font-size: 24px;
+    text-align: center;
+    margin-bottom: 20px;
+}
+
+.best_players_cards__cards {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.player-cards {
+    margin-bottom: 80px;
+}
+
+.player-card {
+    width: 180px;
+    height: 250px;
+    border: 1px solid #000000;
+    margin-top: 10px;
```

Comment:
if you style more than one margin side use shorthad for margin in this case 
margin: 10px 20px 40px; 
more about short hands for margin https://www.w3schools.com/css/css_margin.asp

Reply:
use shorthands for bellows margins